src/smc-webapp/project_page.cjsx

```diff
@@ -161,21 +161,19 @@ FreeProjectWarning = rclass ({name}) ->
         "#{name}" :
             free_warning_extra_shown : rtypes.bool
             free_warning_closed      : rtypes.bool
+        projects :
+            get_total_project_quotas : rtypes.func
 
     propTypes :
         project_id : rtypes.string
-
-    shouldComponentUpdate : (nextProps) ->
-        return @props.free_warning_extra_shown != nextProps.free_warning_extra_shown or
-            @props.free_warning_closed != nextProps.free_warning_closed or
-            @props.project_map?.get(@props.project_id)?.get('users') != nextProps.project_map?.get(@props.project_id)?.get('users')
```

Comment:
Why did you remove this? It prevents the component from updating when the `project_map` changes arbitrarily.